Install.md

```diff
@@ -1,321 +1,224 @@
-      ````
- * Node.js
-   - `sudo apt-get install node.js`
-   - `sudo apt-get install npm`
-    
-### (2) Data base (a): start a postgres service
- * OSX :
-   ```` 
-   (root)$ initdb /usr/local/var/postgres -E utf8
-   (root)$ postgres -D /usr/local/var/postgres &` (for launching a server)
-   ````
- * Ubuntu : 
-   ````
-   (root)$ service postgresql start
-   ````
-### (3) Data base (b): enable remote access to PostgreSQL database server via TCP/IP 
-The reference for this step [is here](https://www.cyberciti.biz/tips/postgres-allow-remote-access-tcp-connection.html))
- * Change user to postgres : sudo su postgres
- * Enable lient authentication : vim /etc/postgresql/9.6/main/pg_hba.conf
- * Add the following line to this file (host) : all all 127.0.0.1/24 trust
- * Open PostgreSQL config file : vim /etc/postgresql/9.6/main/postgresql.conf
-  * Change listen_adresses and port to the following :
-  ````
-      # - Connection Settings -
-
-      listen_addresses = 'localhost'          # what IP address(es) to listen on;
-                                              # comma-separated list of addresses;
-                                              # defaults to 'localhost'; use '*' for all
-                                              # (change requires restart)
-      port = 5432                             # (change requires restart)
-  ````
-
-  * switch back user : `exit`
-  * restart psql : `sudo service postgresql restart`  
-
-### (4) Data base (c): Create an empty database
- * OSX:
-   ````
-   createdb bozo
-   ````
-
-* Ubuntu ([reference](https://wiki.debian.org/PostgreSql)): 
-   ````
-   (root)$ sudo adduser db_user          # that is at OS level
-           ...
-   (root)$ sudo su postgres
-   (postgres)$ createuser db_user        # this account (same name as above) is at PostGreSql level  
-   (postgres)$ createdb -O db_user bozo  # creates a new citydb_v3 database
-   (postgres)$ exit
-   ````
- 
-### (5) Data base (d): Add postgis extension and create a city table
-```` 
-(postgres)$ psql bozo
-  bozo=# create extension postgis;
-  bozo=# \q`  (or use CTRL d equivalently)
-````    
-In the following interactions with the above created data base it is advised to `sudo su db_user`
-```` 
-  (db_user)$ psql bozo
-    bozo=# create table lyon(gid serial primary key, geom GEOMETRY('POLYHEDRALSURFACEZ', 3946));
-    bozo=# alter user dbuser with superuser;
-    bozo=# \q  (or use CTRL d equivalently)
+This document first explains how to install and configure all the components
+described in the architecture diagram above.
+
+It then explains how to use and run all the components for a 3D and a 3D+time
+usecases.
+
+## Install notes
+
+### Install 3DCityDB
+
+If you need to install a 3DCityDB database, follow the instructions described
+[here](Install/Install3DCityDB.md).
+
+### Install UDV-server
+
+[UDV-server](https://github.com/MEPP-team/UDV-server) is a collection of tools
```

Comment:
You shouldn't have line breaks inside a markdown sentence (otherwise the presented layout will a line break too)....

Reply:
I don't think it will, you need to skip a line to have a line break inside the presented layout . I breaked lines between sentences all along the document in order to try to comply with the 80 columns limit. I can change that if you think its confusing when reading the script ?

Reply:
Oh well, within my Atom markdown viewer a break a source line is reflected by a breaks in the interpreted document. I thus made the corrections I suggested and commited the result.